whisper-playground/Containerfile

```diff
@@ -0,0 +1,14 @@
+FROM quay.io/centos/centos:stream8
```

Comment:
Unfortunately, UBI did not support the `ffmpeg` package since it requires the SDL2 library as a dependency. Therefore had to use Centos instead. 
Related issues with UBI:
https://stackoverflow.com/questions/73597789/how-to-install-ffmpeg-on-ubi-docker-images

Reply:
resolved this by installing the static ffmpeg binary instead